<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightsEditorDialog</class>
 <widget class="QWidget" name="LightsEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lights Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
    background-color: #303645;
    color: rgb(92,102,113);
    font-weight: 550;
    padding: 5px 0px;
    border-radius: 0px;
}
QPushButton {
    background-color: rgb(35,35,37);
    border: none; color: rgb(107,117,128);
    padding: 1px 0px; text-align: center;
    font-size: 9pt; line-height: 23px; margin: 0;
}
QPushButton:pressed { background-color: rgb(68,93,132); }
QPushButton:hover  { background-color: rgb(35,35,36); }

#colorSwatch {
  padding: 0;
  border: 0px solid #555A66;
  background-color: #3a4150;
}

QComboBox {
    background-color: #282b2e;
    border: none;
    border-radius: 0px;
    padding-left: 5px;
    color: rgb(107, 117, 128);
}

QComboBox::down-arrow {
    image: url(:/images/arrow_down.png);
    width: 8px;
    height: 8px;
    border-radius: 0px;
}

QComboBox::drop-down {
    border: none;
    border-radius: 0px;
}

QComboBox::popup {
    border: none;
    background-color: #282b2e;
    padding: 0px;
    overflow: hidden;
    border-radius: 8px;
}

QComboBox QAbstractItemView {
    background-color: #282b2e;
    color: white;
    border: none;
    border-radius: 0px;
    border-bottom-left-radius: 4px;
    border-bottom-right-radius: 5px;
}

QComboBox::indicator {
    width: 8px;
    height: 8px;
    background-color: transparent;
}

QSlider::groove:horizontal {
    height: 6px;
    background: #2b2f36;
    border-radius: 3px;
    margin: 0 8px;
}
QSlider::handle:horizontal {
    width: 14px;
    height: 14px;
    border-radius: 7px;
    background: #6b7580;
    margin: -4px -8px;
}

QSplitter {
    background: transparent;
    border: 0px;
    margin: 0px;
    padding: 0px;
}

QSplitter::handle {
    background: transparent;
    border: 0px;
    margin: 0px;
    padding: 0px;
}
QSplitter::handle:horizontal { width: 0px; }
QSplitter::handle:vertical   { height: 0px; }

QSlider::handle:horizontal:hover { background: #7e8a96; }
QSlider::sub-page:horizontal { background: #445066; border-radius: 3px; }
QSlider::add-page:horizontal { background: #2b2f36; border-radius: 3px; }

QPushButton {
    background-color: rgb(35, 35, 37);
    border: none;
    color: rgb(127, 137, 148);
    padding: 1px 0px;
    text-align: center;
    text-decoration: none;
    font-size: 9pt;
    line-height: 23px;
    margin: 0px 0px;
}

QPushButton:pressed {
    background-color: rgb(68, 93, 132);
}

QPushButton:hover {
    background-color: rgb(35, 35, 36);
}

QPushButton:checked {
    background-color: rgb(68, 93, 132);
    color: white;
}

QListWidget {
    border: 0px;
    outline: none;
    background: transparent;
}

QListWidget::item {
    border-radius: 0px;
    margin-left: 1px;
    margin-right: 1px;
    padding: 0px 8px;
}

QListWidget::item:selected {
    background: rgba(60, 120, 215, 180);
    color: white;
}

QListWidget::item:selected:!active {
    background: rgba(60, 120, 215, 160);
    color: white;
}

QListWidget::item:hover {
    background: rgba(60, 120, 215, 90);
}

QListView::item {
    border-radius: 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutRoot">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitterMain">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>

     <!-- LEFT PANEL -->
     <widget class="QWidget" name="leftPanel">
      <layout class="QVBoxLayout" name="leftLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>

       <item>
        <widget class="QFrame" name="leftHeaderFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="leftHeaderLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lightsTitleLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred"/>
            </property>
            <property name="text">
             <string>Light List</string>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item>
        <widget class="QListWidget" name="lightList">
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
        </widget>
       </item>

       <item>
        <widget class="QFrame" name="leftFooterFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <layout class="QHBoxLayout" name="leftFooterLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="leftFooterSpacerLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred"/>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bottomAddLightButton">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bottomRemoveLightButton">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

      </layout>
     </widget>

     <!-- RIGHT PANEL -->
     <widget class="QWidget" name="rightPanel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred"/>
      </property>
      <property name="minimumSize">
       <size>
        <width>420</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>420</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="rightLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>

       <item>
        <widget class="QFrame" name="rightHeaderFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <layout class="QHBoxLayout" name="rightHeaderLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="toggleLeftButton">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Show/Hide light list</string>
            </property>
            <property name="text">
             <string>â˜°</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lightPropsTitleLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred"/>
            </property>
            <property name="text">
             <string>Light Properties</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

       <item>
        <widget class="QFrame" name="propsFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <layout class="QGridLayout" name="propsGrid">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>1</number>
          </property>

          <item row="0" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameEdit"/>
          </item>

          <item row="1" column="0">
           <widget class="QLabel" name="enabledLabel">
            <property name="text">
             <string>Enabled</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="enabledCheckBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>

          <item row="2" column="0">
           <widget class="QLabel" name="typeLabel">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="typeCombo">
            <item>
             <property name="text"><string>Directional</string></property>
            </item>
            <item>
             <property name="text"><string>Point</string></property>
            </item>
            <item>
             <property name="text"><string>Spot</string></property>
            </item>
           </widget>
          </item>

          <item row="3" column="0">
           <widget class="QLabel" name="colorLabel">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="colorLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="colorPreviewButton">
              <property name="objectName">
               <string>colorSwatch</string>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="colorPickButton">
              <property name="minimumSize">
               <size>
                <width>26</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>C</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>

          <item row="4" column="0">
           <widget class="QLabel" name="intensityLabel">
            <property name="text">
             <string>Intensity</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSlider" name="intensitySlider">
            <property name="minimum"><number>0</number></property>
            <property name="maximum"><number>200</number></property>
            <property name="value"><number>50</number></property>
            <property name="orientation"><enum>Qt::Orientation::Horizontal</enum></property>
           </widget>
          </item>

          <item row="5" column="0">
           <widget class="QLabel" name="rangeLabel">
            <property name="text">
             <string>Range</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSlider" name="rangeSlider">
            <property name="minimum"><number>0</number></property>
            <property name="maximum"><number>500</number></property>
            <property name="value"><number>100</number></property>
            <property name="orientation"><enum>Qt::Orientation::Horizontal</enum></property>
           </widget>
          </item>

          <item row="6" column="0">
           <widget class="QLabel" name="spotInnerLabel">
            <property name="text">
             <string>Spot Inner</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSlider" name="spotInnerSlider">
            <property name="minimum"><number>0</number></property>
            <property name="maximum"><number>100</number></property>
            <property name="value"><number>60</number></property>
            <property name="orientation"><enum>Qt::Orientation::Horizontal</enum></property>
           </widget>
          </item>

          <item row="7" column="0">
           <widget class="QLabel" name="spotOuterLabel">
            <property name="text">
             <string>Spot Outer</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSlider" name="spotOuterSlider">
            <property name="minimum"><number>0</number></property>
            <property name="maximum"><number>100</number></property>
            <property name="value"><number>80</number></property>
            <property name="orientation"><enum>Qt::Orientation::Horizontal</enum></property>
           </widget>
          </item>

          <item row="8" column="0">
           <widget class="QLabel" name="affectRasterLabel">
            <property name="text">
             <string>Affect Raster</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QCheckBox" name="affectRasterCheckBox">
            <property name="text"><string/></property>
            <property name="checked"><bool>true</bool></property>
           </widget>
          </item>

          <item row="9" column="0">
           <widget class="QLabel" name="affectRtLabel">
            <property name="text">
             <string>Affect RT</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QCheckBox" name="affectRtCheckBox">
            <property name="text"><string/></property>
            <property name="checked"><bool>true</bool></property>
           </widget>
          </item>

          <item row="10" column="0">
           <widget class="QLabel" name="castShadowsLabel">
            <property name="text">
             <string>Cast Shadows</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QCheckBox" name="castShadowsCheckBox">
            <property name="text"><string/></property>
            <property name="checked"><bool>true</bool></property>
           </widget>
          </item>

         </layout>
        </widget>
       </item>

       <item>
        <spacer name="rightBottomSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>

       <item>
        <widget class="QFrame" name="rightFooterFrame">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <layout class="QHBoxLayout" name="rightFooterLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="rightFooterSpacerLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed"/>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed"/>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>

      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
